src/test/java/de/tum/in/www1/artemis/DatabaseQueryCountTest.java

```diff
@@ -38,10 +38,11 @@ void testGetAllCoursesForDashboardRealisticQueryCount() throws Exception {
             var userCourses = request.getList("/api/courses/for-dashboard", HttpStatus.OK, Course.class);
             log.info("Finish courses for dashboard call for multiple courses");
             return userCourses;
-        }).hasBeenCalledAtMostTimes(15);
+        }).hasBeenCalledAtMostTimes(16);
```

Comment:
This test currently fails. It looks like `.hasBeenCalledAtMostTimes(9);` below has to be incremented, too.